db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -0,0 +1,20 @@
+class AddStiToContainerImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_images, :type, :string
```

Comment:
I tried this split but it doesn't work, The `up` method is not called for some reason that is beyond me. Can you explain the benefits of that approach? How exactly should this be done?